src/main/java/ch/njol/skript/conditions/CondIsBlockRedstonePowered.java

```diff
@@ -25,25 +25,40 @@
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
 
 @Name("Is Block Redstone Powered")
-@Description("Checks if a block is powered by redstone")
+@Description("Checks if a block is (indirectly) powered by redstone")
 @Examples({"if clicked block is redstone powered:",
-	"\tsend \"This block is well-powered by redstone!\""})
+	"\tsend \"This block is well-powered by redstone!\"",
+	"if clicked block is indirectly redstone powered:",
+	"\tsend \"This block is indirectly redstone powered.\""})
 @Since("2.5")
 public class CondIsBlockRedstonePowered extends PropertyCondition<Block> {
 	
 	static {
-		register(CondIsBlockRedstonePowered.class, "redstone powered", "blocks");
+		register(CondIsBlockRedstonePowered.class, "(0¦|1¦indirectly) redstone powered", "blocks");
```

Comment:
If you put brackets ``()`` in ``register()``, it means that that's a mandatory field, i.e. ``redstone powered`` will no longer be a valid condition with your change. I think you meant to use square brackets ``[]`` instead? Those indicate that it's an optional phrase.

Reply:
The way they did it is valid, since the options would be "" and "indirectly"

Maybe redstone should be optional though

Reply:
Really? Didn't know that about parse marks... haha

Reply:
Thank you for your reviews!
Concerning making `redstone` optional, I was advised to use `redstone power` to avoid conflicts (https://github.com/SkriptLang/Skript/pull/3091#discussion_r448349996).